### PR TITLE
9296 nameX - self-serve statistics for examiners

### PR DESCRIPTION
**Pull request by eve-git**
*Issue #:*
https://github.com/bcgov/entity/issues/9296

*Description of changes:*

- Add 'My Stats' checkbox and Total Name Examined to the stats page

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: client/src/components/application/Stats/Stats.vue [hunk 1]
```diff
@@ -131,7 +141,8 @@ export default {
     }
   },
   mounted() {
-    if (this.$refs.numberinput) this.$refs.numberinput.focus()
+    this.getPagedStats();
+    if (this.$refs.numberinput) this.$refs.numberinput.focus();
```
Comment (severinbeauvais) [inline client/src/components/application/Stats/Stats.vue:145]: Not a big deal but we don't use trailing commas. Funny the linter didn't flag these.

File: client/src/components/application/Stats/Stats.vue [hunk 2]
```diff
@@ -183,6 +194,9 @@ export default {
       let currentPage = '&currentpage=' + this.currentPage
       let pageSize = '&perpage=' + this.perPage
       let url = '/api/v1/requests/stats' + params + currentPage + pageSize
+      if (this.myStats) {
+        url += '&myStats=True'
```
Comment (severinbeauvais) [inline client/src/components/application/Stats/Stats.vue:198]: Where does this URL parameter get consumed?

File: client/src/components/application/Stats/Stats.vue (hunk 1 above)
Comment (ozamani9gh) [inline client/src/components/application/Stats/Stats.vue:145]: remove semicolon.

**Comment by severinbeauvais** [comment]
Eve, the GCP build didn't work because no one was assigned.

/gcbrun

**Comment by severinbeauvais** [comment]
Um, maybe gcbrun isn't configured for this repo.

**Review by ozamani9gh — COMMENTED** [review]
except for semicolons. looks good

File: client/src/components/application/Stats/Stats.vue (hunk 2 above)
Comment (eve-git) [reply client/src/components/application/Stats/Stats.vue:198]: The url is consumed in namex. If myStats=True, userId is a condition of the stats query.

Comment (severinbeauvais) [reply client/src/components/application/Stats/Stats.vue:198]: OK, thanks.

I see `numRecords` is read from the API response on line 219, which answers the overall question 👍

**Review by severinbeauvais — APPROVED** [review]

File: client/src/components/application/Stats/Stats.vue (hunk 1 above)
Comment (severinbeauvais) [reply client/src/components/application/Stats/Stats.vue]: Thanks for the fix.

**Review by ozamani9gh — APPROVED** [review]
looks good to me

**Comment by severinbeauvais** [comment]
This feature will have to ported to "main" branch, right?

**Pull request merged**